spec/dfns.txt

```diff
@@ -0,0 +1,22 @@
+# Add a new DFN parameter set to MF6
```

Comment:
In mf6 `dfns.txt` specifies the subset of DFNs supported by IDM. I think we don't need a similar filtering mechanism here but could be wrong?

Reply:
Probably not but I thought maybe it would be convenient temporarily to develop with a restricted set- if we want them all from the outset we can do away with that.  Same can be said for the toml files that are being generated-  there is some processing that it happening (similar to mf6), but those should be removed or updated as appropriate for this project.

Reply:
cool that sounds good to me. I'm fine with keeping it

Reply:
Also, common.dfn is an outlier and I don't think currently will convert.  I think it needs special handling.